analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/descriptions/TypeTypeFilter.java

```diff
@@ -2,13 +2,28 @@
 
 import java.util.Objects;
 
+/**
+ * Helper class to filter the types that need to be serialized (all excluding
+ * {@link TypeType#CLASS}).
+ */
 public class TypeTypeFilter {
 
+  /**
+   * Filter out non-default values (so skip "Type":0).
+   *
+   * @param obj TypeType object
+   * @return True if obj is a non-default value, False if obj is {@link TypeType#CLASS}.
```

Comment:
Are we sure about this? `TypeType.CLASS` is the default, it is `instanceof TypeType` and it is `Objects.equals` to itself. So I would expect this to return `true` for the default (which should be dropped), `false` for the non-default (which should be kept). 

See also the doc from the `@JsonInclude` annotation:

> Whether value if is to be included or not is determined by calling Filter's equals(value) method: if it returns true value is NOT included (it is "filtered out"); if false value IS included ("not filtered out").